README.md

```diff
@@ -36,14 +37,14 @@ You can set the tasks that the crawler runs in `crawler.js`.
 
 ## Testing
 ```sh
-npm test
+gulp test
```

Comment:
`npm test` is agnostic and the default test cmd in the node ecosystem (that's why npm has a `test` cmd after all). `gulp test` is bound to a specific build system. Maybe we decide to switch back to grunt tomorrow? :stuck_out_tongue_winking_eye: